### PR TITLE
chore: splitting sample into single files

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Chore: Splitting one big file of Detect.java into individual java files and their tests.

re-formatted samples according to new SAMPLE_FORMAT

### CODE REVIEW

File: vision/cloud-client/src/main/java/com/example/vision/snippets/DetectCropHints.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 Google Inc.
```
Comment (lesv) [inline vision/cloud-client/src/main/java/com/example/vision/snippets/DetectCropHints.java:2]: This probably needs to be updated.  Most likely to 2020 Google LLC

**Comment by munkhuushmgl** [comment]
@gguuss I do not know what this means:
```
[14:44:58][ERROR] Found unpermitted build inputs
```

File: vision/cloud-client/src/main/java/com/example/vision/snippets/DetectCropHintsGcs.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2017 Google Inc.
```
Comment (lesv) [inline vision/cloud-client/src/main/java/com/example/vision/snippets/DetectCropHintsGcs.java:2]: 2020?

File: vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Google Inc.
```
Comment (lesv) [inline vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java:2]: 2020

File: vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void testLandmarksUrl() throws Exception {
+    // Act
+    String uri =
```
Comment (lesv) [inline vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java:55]: `String.format()` ??

File: vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+    // Assert
+    String got = bout.toString().toLowerCase();
+    assertThat(got).contains("palace of fine arts");
+  }
+
+  @Test
+  public void testLandmarksGcs() throws Exception {
+    // Act
+    DetectLandmarksGcs.detectLandmarksGcs("gs://" + ASSET_BUCKET + "/vision/landmark/pofa.jpg");
```
Comment (lesv) [inline vision/cloud-client/src/test/java/com/example/vision/DetectLandmarksGcsTest.java:67]: `String.format()` ??

**Comment by lesv** [comment]
@munkhuushmgl I don't think the lint issue is yours.  @kurtisvg  is doing some stuff

**Comment by munkhuushmgl** [comment]
> What's going on? We always expect some kind of message for PR's. Especially for something this big.
> 
> Why are the copyrights all 2017 and for Google Inc. rather than 2020 Google LLC ?

Sorry about the confusion, This was part of my clean up tasks. I will update the message

**Comment by munkhuushmgl** [comment]
> _No description provided._

**Pull request merged**